internal/registry/client_test.go

```diff
@@ -49,7 +49,9 @@ func TestSetupClient(t *testing.T) {
 
 			t.Setenv("DOCKER_CONFIG", ".")
 
-			SetupClient(repository)
+			if err := SetupClient(repository); err != nil {
+				t.Error(err)
```

Comment:
I think this should be t.Fatal instead. If this fails, there's no need to continue in the test.